keep/api/alert_deduplicator/alert_deduplicator.py

```diff
@@ -38,6 +38,9 @@ def is_deduplicated(self, alert: AlertDto) -> bool:
         alert_deduplicate = (
             True if get_alert_by_hash(self.tenant_id, alert_hash) else False
         )
+        if alert_deduplicate:
+            self.logger.info(f"Alert {alert.id} is deduplicated {alert.source}")
```

Comment:
```suggestion
            self.logger.debug(f"Alert {alert.id} is deduplicated {alert.source}")
```

Reply:
I've put "info" on purpose since this is the way to count how many deduplication are there, the long term should be add some other metric and then move it to debug